src/main/scala/io/scalac/tezos/translator/Boot.scala

```diff
@@ -70,16 +70,18 @@ object Boot {
         adminEmail        <- Future.fromTry(EmailAddress.fromString(cfg.email.receiver))
         routes            = new Routes(email2SendService, libraryService, userService, MMTranslator, log, cfg.reCaptcha, adminEmail)
         binding           <- Http().bindAndHandle(routes.allRoutes, host, port)
+        _                 <- Future.successful(StdIn.readLine())
```

Comment:
it is safe to create everything other as a successful?

Reply:
discussed on Slack